mewarpx/mewarpx/emission.py

```diff
@@ -780,16 +767,17 @@ def __init__(self, conductor, T, xmin=None, xmax=None,
                 Default mwxrun.ymax.
             transverse_fac (float): Scale the transverse energy distribution by
                 this factor. Default 1. See
-                :func:`metools.runtools.get_velocities` for details.
+                :func:`mewarpx.util.get_velocities` for details.
             kwargs (dict): Any other keyword arguments supported by the parent
                 Emitter constructor (such as "use_Schottky" or
                 "emission_type").
         """
         # Default initialization
+        if not T:
```

Comment:
I prefer `if T is None`, that way if someone sets `T=0` it won't evaluate as False and be overwritten. (It'll break other things, but in principle...)